src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ChangesView.cs

```diff
@@ -31,11 +31,7 @@ public override void InitializeView(IView parent)
         {
             base.InitializeView(parent);
             tree.InitializeView(this);
-        }
 
-        public override void OnEnable()
-        {
-            base.OnEnable();
```

Comment:
Probably don't want to just to this always at initialize, since it calls Repository Refresh and all views get initialized. The logic of doing this on enable is correct, it's just that OnEnable might get called before the Repository instance is actually set, which means that we need to also do this whenever OnGUI is called. #91 introduces some methods that get called to update handlers and data, and the logic then becomes like this: https://github.com/github-for-unity/Unity/blob/3de69a70c0021b26e314b70116d111871e2e07ef/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs#L50-L86

I could have sworn that that PR had already landed in master, really confused how it's not there yet, so that needs to land in order for these refresh bugs to get fixed.